wolnut/__main__.py

```diff
@@ -146,4 +146,9 @@ def main():
 
 
 if __name__ == "__main__":
+    logging.basicConfig(
+        level=logging.INFO,
+        format='%(asctime)s [%(levelname)s] %(message)s',
+    )
```

Comment:
I would move this into the `main` function and then the `project.script` can be `wolnut.__main__:main`, and you don't need the `entrypoint` in `__init__`.

Reply:
I don't know if that's a good idea. One may want to run `main` programatically and messing with their logging config is bad.

Reply:
The main issue for me is the duplication with the entrypoint function in `__init__`.

Reply:
I see. We could extract it to a function like `setup_logging`. But I didn't do it because it's just one line of code.

Reply:
Isn't the reason `if __name__ == '__main__'` exists to allow `__main__` to be imported without executing its code? It's not there because the file is named `__main__`, in fact, the file is `main.py` in the repo.

But yeah we could have the `main` and `entrypoint` both in the same file. In my opinion it would be confusing have both in the `main.py` file because they essentially do the same thing, but we could do it.


The reason I think my approach is less confusing is because currently they do the same thing but there's a distiction between the two: one is the **package** entrypoint (`__main__.py:main`) and the other is the **script** entrypoint (`wolnut:entrypointmain.py`)

Reply:
We could also wait for @hardwarehaven's take on this.

Reply:
Inside `__main__.py`, `__name__ == '__main__'` is always true. So you can't import from it without running the code. I think.

Personally, I would want `python -m wolnut` and `wolnut` to do exactly the same thing. It would be very confusing if they are different in any way. I can't imagine any situation where you'd want something else.

Reply:
Actually, `__name__` [would be set to `wolnut`](https://docs.python.org/3/library/__main__.html#name-main) if doing `from wolnut import __main__`. And they do the same thing, the difference is how they run the program. `wolnut` calling `entrypoint` and `python -m` just `python wolnut/__main__.py` (or something like that).

Like I said, we can have the `entrypoint` function in `__main__` and call it inside the if.

Reply:
Just tested it and it looks like the current approach raises a RuntimeWarning when running `python -m`. Looks like we can't import `from __main__` inside `__init__`.

I think I should rename `__main__.py` to `cli.py`, and in the `__main__.py` just have "from .cli import main" (or entrypoint). What do you think?

Reply:
Yes. That's good for me.